tests/suites/0_stateless/12_0000_insert_into_select2.py

```diff
@@ -30,3 +30,7 @@
 """
 
 client1.run(sqls)
+stdout, stderr = client1.run_with_output("select * from system.metrics")
+assert stdout is not None
```

Comment:
`print(stdout)` will show in the result file?

Reply:
yes, but we should use `print(stdout, end = '')` to ignore the Newline.